src/row.rs

```diff
@@ -134,7 +134,7 @@ mod tests {
     fn test_some_functions() {
         let cells = vec!["one", "two", "three"];
         let mut row = Row::new();
-        for cell in cells.iter() {
+        for cell in &cells {
```

Comment:
I'm generally a big fan of the `.iter()`, `.iter_mut()` and `.into_iter()` syntax, as it makes the actual borrowing/ownership rules of the iterated values much more clear, explicit and they all look uniformly.

At least it makes them more clear for me :D.
I would like to ask you to revert this specific commit, but thanks for the suggestion anyway!

Reply:
no problem. I've dropped that commit